### ISSUE TITLE
 IOU - "Unexpected error.." message appears when employee click Submit

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#34259]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.24-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

**Issue found when executing PR** https://github.com/Expensify/App/pull/33811
## Action Performed:
1. Log in to ND as the employee
2. Go to the workspace chat > + > Request money
3. Create a manual request
4. Create Scan request
5. Click on the expense preview
6. Click Submit while scan is still in progress

## Expected Result:
There should be no error message when user click Submit

## Actual Result:
"Unexpected error Please try again later" message appears when employee click submit

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details>  

https://github.com/Expensify/App/assets/78819774/7a68e1d2-3fb4-4f41-ab22-126a236cebcd

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by shubham1206agra** [comment]
Umm.. This is expected. You can't submit request when scanning request is underway. For showing meaningful, you need to configure BE to give correct message.

**Comment by NikkiWines** [comment]
🤔 I don't even see the option to submit the request while it's still scanning. But yes, we should handle any attempts to do that with a better error message. 

<img width="1376" alt="image" src="https://github.com/Expensify/App/assets/16747822/f775656b-faa4-45ee-a16d-e0f379fda046">

@lanitochka17 are there more specific steps to be able to see the `Submit` button in this case?

**Comment by NikkiWines** [comment]
Anyway, looking at the testing videos in https://github.com/Expensify/App/pull/33811 it looks like this is actually the intended behavior and the testing steps are just unclear. Double checking against the issue that PR resolves we can see the [accepted proposal](https://github.com/Expensify/App/issues/32374#issuecomment-1837058926) shows this exact flow as it's reproduced here. 

TLDR; expected behavior (as @shubham1206agra said), not a blocker. Closing!
